README.md

```diff
@@ -442,3 +441,242 @@ receiver.addHandler(new ExampleReceiver() {
+    public void handle(JSObject jsObject) {
+        // handle result
+    }
+});
+
+// calls method and handles callback by handler created with @Handler annotation
+// (see the example of generating handlers)
+proxy.callDefaultInterfaceMethod(new JSTPOkErrorHandler() {
+    @Override
+    public void onOk(List<?> args) {
+        // handle data
+    }
+
+    @Override
+    public void onError(Integer errorCode) {
+        // handle error code
+    }
+});
+
+// calls method and handles callback by executable handler (see the example of
+// executable handler)
+proxy.callDefaultInterfaceMethod(new ExecutableHandler(executor) {
+    @Override
+    public void run() {
+        // handle message received
+    }
+});
+
+// sends event with specified parameters
+proxy.sendEvent("myParam");
+```
+
+
+##### Call handler
+
+You can process incoming `call` packets by setting `call` handlers for called
+methods, just like if you did it with `Connection` directly. You can set `call`
+handler by `setCallHandler()` method, and remove your handler by
+`removeCallHandler()` method. See the examples:
+
+```java
+// sets manual call handler for "methodName" of "interfaceName"
+proxy.setCallHandler("interfaceName", "methodName", new ManualHandler() {
+    @Override
+    public void handle(JSObject jsObject) {
+        // handle object
+    }
+});
+
+// sets executable call handler for "methodName" of "interfaceName" (see the
+// example of executable handler)
+proxy.setCallHandler("interfaceName", "methodName",
+        new ExecutableHandler(executor) {
+    @Override
+    public void run() {
+        // handle message received
+    }
+});
+```
+
+If we want to use handler created with `@Handler` annotation like this
+
+```java
+@Handler
+public interface OkErrorHandler {
+
+    @NotNull
+    @Object("ok")
+    void onOk(List<?> args);
+
+    @NotNull
+    @Object("error")
+    void onError(@Array(0) Integer errorCode);
+}
+```
+
+We can use generated `JSTPOkErrorHandler` as a call handler for proxy:
+
+```java
+proxy.setCallHandler("interfaceName", "methodName", new JSTPOkErrorHandler() {
+    @Override
+    public void onOk(List<?> args) {
+        // handle data
+    }
+
+    @Override
+    public void onError(Integer errorCode) {
+        // handle error code
+    }
+});
+```
+
+##### Event handler
+
+You can process incoming `event` packets by adding `event` handlers, just like
+if you did it with `Connection` directly. You can add `event` handler by
+`addEventHandler()` method, and remove your handler by `removeEventHandler()`
+method. See the examples:
+
+```java
+// adds manual event handler for "eventName" of "interfaceName"
+proxy.addEventHandler("interfaceName", "eventName", new ManualHandler() {
+    @Override
+    public void handle(JSObject jsObject) {
+        // handle object
+    }
+});
+
+// adds executable handler for "eventName" of "interfaceName" (see the example
+// of executable handler)
+proxy.addEventHandler("interfaceName", "eventName",
```

Comment:
I think it's better to add example of handler with ExecutableHandler as its base, not just ExecutableHandler itself.